### ISSUE TITLE
How does this PaaS stand out from TADS Boilerplate?

### ISSUE THREAD

**Issue by Cloufish** [sergioisidoro/honey-swarm#16]
I noticed the following differences:
- Portainer and Traefik Agents installed additionally

but any more? xd 
It would be nice to have a section dedicated for your features

**Comment by sergioisidoro** [comment]
Hi, 

Thanks for the feeedback, but I think that's pretty much it :)

The main difference is that I wanted to make a ready made Portainer installation, and Portainer App templates, and for that I needed to make a couple of decisions - eg committing to a ingress (Traefik), creating shared docker network for services to be public.

Also I made some attempt at creating a default set of security and hardening that fits maybe 80% of people, but not all

I think these decisions depart a bit from the generalist TADS philosophy that probably need to fit a much wider audience. 

TLDR - TADS is supposed to be a infrastructure management project, I wanted this to be more of a PaaS with Portainer+Traefik :)
